src/types.ts

```diff
@@ -481,11 +481,15 @@ export interface AirshipConfig {
      */
     itunesId?: string;
 
-
     /**
      * If set to `true`, the SDK will use the preferred locale. Otherwise it will use the app's locale.
      */
     useUserPreferredLocale?: boolean;
+
+    /**
+     * Allows the WebViews to be inspected in Safari.
+     */
+    isWebViewInspectionEnabled?: boolean;
```

Comment:
this needs to be wired up in the mobile framework proxy

Reply:
Added in this PR : https://github.com/urbanairship/airship-mobile-framework-proxy/pull/46